### PR TITLE
Updating Brazilian Portuguese Translation

### PR DESCRIPTION
**Pull request by altiereslima**
Update for version 0.60

### CODE REVIEW

**Comment by TheCycoONE** [comment]
I'm guessing we don't have any reviewers for brazilian portuguese so we'll have to take it on faith that your translations are more accurate than what we had before. That said your first commit also does a significant reformatting of brazilian_portuguese.lua, moving a lot of prompts around.

Could you walk us through how you arrived at that layout, what you corrected vs. what you added, etc.?

**Comment by altiereslima** [comment]
This translation was based on the current english.lua layout.

File: CorsixTH/Lua/languages/brazilian_portuguese.lua [hunk 1]
```diff
@@ -475,12 +476,12 @@ new_game_window = {
...
-  difficulty = "Selecione o nível de dificuldade do jogo.",
+  difficulty = "Selecione o nível de dificuldade do jogo",
   easy = "Se você é novo em jogos de simulaçäo esta é a opçäo para você",
   medium = "Este é o meio do campinho se você está inseguro do que deve escolher",
   hard = "Se você está acostumado com este tipo de jogo e quer mais desafios, escolha esta opçäo",
   tutorial = "Se você precisa de ajuda para começar a jogar, escolha esta opçäo",
-  start = "Iniciar o jogo com a configuraçäo selecionada.",
+  start = "Iniciar o jogo com a configuraçäo selecionada",
   cancel = "Ah, eu näo queria começar um novo jogo!",
 }
```
Comment (MarkL1961) [inline CorsixTH/Lua/languages/brazilian_portuguese.lua]: checking actual English text in google this should be Começar for start and dificuldade for difficulty.
I don't understand why one word in English file becomes more detailed and maybe won't fit in the space in the translation

Comment (MarkL1961) [inline CorsixTH/Lua/languages/brazilian_portuguese.lua]: checking actual English text in google this should be Começar for start and dificuldade for difficulty.
I realise here that all you are doing is correcting what is already in the file by removing punctuation,  I don't understand why one word in English file becomes more detailed (and maybe won't fit in the space) in the translation

**Comment by TheCycoONE** [comment]
Do you know enough about git to squash your commits, or would you like me to?

**Comment by altiereslima** [comment]
I Don't Know Enough

**Comment by TheCycoONE** [comment]
Note that whenever I rebase this it's going to cause you problems, so let me know when you're done with the updates :)

**Comment by altiereslima** [comment]
Ok TheCycoONE, finished updates. :)
Thank you.

**Comment by TheCycoONE** [comment]
I'll give it a couple days in case you change your mind

**Comment by TheCycoONE** [comment]
Are those last changes still good translations? Because I think we have to fix the dialog box problem anyway, but I don't know when that will be.

**Comment by altiereslima** [comment]
Yes, are those last changes still good translations

**Pull request merged**